language/llama2-70b/processorca.py

```diff
@@ -67,8 +83,11 @@ def load_parquet(self) -> pd.DataFrame:
         print(f"Tokenizing input")
         df.rename(columns={'response': 'output'}, inplace=True)
         df['input'] = df.apply(format_llama_input, axis=1)
-        df['tok_input'] = df['input'].apply(lambda x: llama_tokenizer(x)['input_ids'] if isinstance(x, str) else [])
-        df['tok_output'] = df['output'].apply(lambda x: llama_tokenizer(x)['input_ids'] if isinstance(x, str) else [])
+
+        input_tokenizer = partial(_tokenize_helper, llama_tokenizer=llama_tokenizer)
```

Comment:
Dumb question: does this make the input 1 token longer than expected (longest 1025)?

Reply:
No, the appending of the token happens when we load the parquet, and the seqlen filtering only happens after.

I should mention here: https://github.com/nv-alicheng/inference/blob/dev-alicheng-gpu-bringup/language/llama2-70b/processorca.py#L109-L110 That we are doing `<` instead of `<=`, so I think it's actually doing 1023 not 1024... In practice I don't think that matters though.